.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug Report
+about: Report a bug encountered while operating Osiris
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->
```

Comment:
```suggestion
<!-- Please use this template while reporting a bug and provide as much info as possible. This would help us address your bug in a timely manner. Thanks!-->
```

Reply:
That's my opinion. I'd still leave it to you to decide @krancour